src/tutorial/meshvis.md

```diff
@@ -55,47 +53,56 @@ Rc=L/10;
 ```
 
-Gmsh will use the obtained geometry for generating the mesh. However, without additional specifications, we cannot impose boundary conditions without any attributes assigned to the boundaries. Different attributes can be assigned to the volumetric part of the mesh for using different material coefficients within the domain. Here, however, we will use only a single attribute, as the first example uses only a single diffusion coefficient. 
+Gmsh uses the obtained geometry for generating the mesh. However, without additional specifications, we cannot impose boundary conditions without any attributes assigned to the boundaries. Different attributes can be assigned to the volumetric part of the mesh for using different material coefficients within the domain. Here, however, we use only a single attribute, as the first example uses only a single diffusion coefficient.
+
 ```diff
 Physical Volume(1) = {50};
 Physical Surface(1) = {1,6,8};
 
 Mesh.MshFileVersion = 2.2;
 ```
-The first line from the above snippet will define physical volume 1 to coincide with the geometry volume 50, which is the final volume obtained by the boolean operation. The second line will define physical surface 1 to include geometric surfaces {1,6,8}. Finally, the last line specify the the file format. It should be pointed out that MFEM can only read ASCII Gmsh format version 2.2. 
 
+The first line from the above snippet defines physical volume 1 to coincide with the geometry volume 50, which is the final volume obtained by the Boolean operation. The second line defines physical surface 1 to include geometric surfaces {1,6,8}. Finally, the last line specifies the file format. Note that MFEM can only read ASCII Gmsh format version 2.2.
 
 <img style="width:60%" src="../img/gmsh02.png"> 
 <img style="width:60%" src="../img/gmsh03.png">
 
-The generated mesh is shown in the figure above. Careful inspection reveals that the cylindrical surface is not represented well by the linear elements. We can improve the representation by refining the mesh. The Gmsh input file can be downloaded  <a href="../cross_heat.geo">here</a>  and the mesh <a href="../cross_heat.msh">here</a>. The users are strongly encouraged to play with the mesh and to generate finer discretizations for the simulations.  
+The generated mesh is shown in the figures above. Careful inspection reveals that the cylindrical surface is not represented well by the linear elements. We can improve the representation by refining the mesh. We encourage you to play with the mesh and to generate finer discretizations for the simulations.  
+
+You can download the Gmsh input file [here](../cross_heat.geo) and the mesh [here](../cross_heat.msh). For users without access to the Gmsh GUI, a mesh can be generated in a terminal with the following command:
 
-For users without access to the graphical interface of Gmsh, a mesh can be generated in a terminal by the following command: 
 ```diff
 gmsh -3 cross_heat.geo
```

Comment:
Where in the mfem directory (or outside of it) should I be when running this command?